localstack-core/localstack/config.py

```diff
@@ -429,6 +429,10 @@ def in_docker():
 LS_LOG = eval_log_type("LS_LOG")
 DEBUG = is_env_true("DEBUG") or LS_LOG in TRACE_LOG_LEVELS
 
+# PUBLIC: 0 (default)
```

Comment:
Let's introduce this feature as public preview until we have more feedback about how it's working and the naming:

```
# PUBLIC PREVIEW: 0 (default), 1 (preview)
```